README.md

```diff
@@ -1 +1,65 @@
-# labclimate
+labclimate
+==========
+
+Database Table Description (Assuming Prostresql)
+------------------------------------------------
+
+### **data** table:
+
+| Field Name  | Field Type |
+|-------------|------------|
+| node\_id    | bigint     |
+| timestamp   | timestamp  |
+| temperature | double     |
+| humidity    | double     |
+| air\_qual   | double     |
+
+
+### **nodes** table:
+
+| Field Name     | Field Type   |
+|----------------|--------------|
+| node\_id       | bigint       |
+| node\_location | varchar(256) |
+
+
+API Endpoints
+---------------
+
+| Method | Endpoint         | Usage                                          |
+|--------|------------------|------------------------------------------------|
+| GET    | /config          | Gets the client configuration from the server. |
+| POST   | /data/{id}       | Submits climate data to the server.            |
+| POST   | /nodes           | Adds a new node to the server.                 |
+| DELETE | /nodes/{id}      | Deletes a node from the server.                |
```

Comment:
When would this be called? If the node is going offline chances are it is going down due to a powercut in which case this is redundant. Better make the server handle this case instead of an explicit API call, imho.